addons/mail/wizard/mail_compose_message.py

```diff
@@ -1054,12 +1055,22 @@ def _prepare_mail_values_rendered(self, res_ids):
             new_attachment_ids.reverse()
             self.write({'attachment_ids': [Command.set(new_attachment_ids)]})
 
+        # update partner_ids from mentions in body
```

Comment:
Composer should not depend on some specific custom code embed in HTML. Instead do a processing in JS that updates partner_ids accordingly. Server should not have to parse the body, and should always consider it is complete and containing only QWeb directive as dynamic content.

Reply:
Ok, I will do the data process in the JS side then !